internal/core/entities/autoscaling/autoscaling.go

```diff
@@ -87,5 +88,5 @@ type RoomOccupancyParams struct {
 	// ReadyTarget indicates the target percentage of ready rooms a scheduler should maintain.
 	ReadyTarget float64 `validate:"gt=0,lte=0.9"`
 	// DownThreshold indicates the percentage of occupied rooms a scheduler should have to trigger a downscale event.
-	DownThreshold float64 `validate:"gte=0,lte=0.99"`
+	DownThreshold float64 `validate:"gt=0,lt=1"`
```

Comment:
nit: 

```suggestion
	DownThreshold float64 `validate:"gt=0,lte=0.99"`
```

to be the same as `ReadyTarget`